fdbrpc/sim2.actor.cpp

```diff
@@ -536,7 +536,10 @@ class SimpleFile : public IAsyncFile, public ReferenceCounted<SimpleFile> {
 
 	std::string getFilename() const override { return actualFilename; }
 
-	~SimpleFile() override { _close(h); }
+	~SimpleFile() override {
+		_close(h);
+		--openCount;
```

Comment:
Is there a reason why the `openCount` was not previously decreased here? @etschannen might know?

Reply:
I think this was just an oversight when I originally added the TooManyFiles check